### DISCUSSION TITLE
Looking for Users with Access to MEB (ID) Car with SW Version 3.0

### DISCUSSION THREAD

**Discussion by tillsteinbach** [tillsteinbach/VWsFriend#194] [General]
To prepare for the compatibility with the upcoming 3.0 release I'm looking for people with early access to 3.0 SW to report and test compatibility. I guess my ID.3 will only get the update late this year :(

**Comment by Julien4135** [comment]
Hello, I'll let you know in the next few weeks, I should be among the first logically in my garage.

**Reply by tillsteinbach** [reply]
Thanks! There are already first reports coming in. One issue is that there is no data coming in when a secondary user is logged in. Both in VWsFriend and the official app.

**Comment by Julien4135** [comment]
yes because now with the update the PIN code becomes mandatory

**Reply by Julien4135** [reply]
Hi, I have an appointment for the 2.4 update on August 8, normally 3.0 is immediately after so I'll let you know from that date.

**Reply by tillsteinbach** [reply]
I have an appointment the 27th of July, but rumors say there will be no OTA for 3.0 before CW35 and even then it can be two weeks to get it as the car is only asking once a week for updates and it is two parts again.

**Comment by Grogilein** [comment]
My ID.3 Pro Performance is going into the shop this Wednesday (07/20/2022). The windshield will be replaced, the air conditioning repaired and the vehicle will receive the new battery with update to software level 2.4. The subsequent OTA update to 3.0 will probably come quite quickly. I would make my vehicle available for testing VWsFriend.

**Comment by MattiJarvinen** [comment]
The rumour is on the FB that 3.0 OTA will start rolling mid August. There should be new car's with 3.0 already.

**Reply by tillsteinbach** [reply]
There are even rumors the update could start next monday. Yes there are 3.0 cars already on the road and since this week even 3.1 cars should be produced.

**Reply by Julien4135** [reply]
Hi.
Any news from your side?

**Reply by tillsteinbach** [reply]
There are reports from Users with 3.0 that it is working

**Reply by Julien4135** [reply]
Great then. It's great
